recipes/runthis-server/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "runthis-server" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/regro/runthis-server/releases/download/{{ version }}/runthis-server-{{ version }}.tar.gz
+  sha256: 8230b42c3231b555679ea278678c6a6ddaf4a8a606f2b51c9020762f7b9c0525
+
+build:
+  noarch: python
```

Comment:
This can't be noarch if it's using [`scripts`](https://github.com/regro/runthis-server/blob/master/setup.py). Can they be converted to `entry_points`?

Reply:
Oh yeah, right

Reply:
You’ll have to modify the source and make a new release to do it this way